components/library/qt5/patches/zoneinfo.patch

```diff
@@ -0,0 +1,125 @@
+--- qt-everywhere-src-5.15.13/qtwebengine/src/3rdparty/chromium/third_party/icu/source/common/putil.cpp	2023-02-27 19:57:08.000000000 +0100
++++ qt-everywhere-src-5.15.13/qtwebengine/src/3rdparty/chromium/third_party/icu/source/common/putil.cpp.new	2024-06-25 20:47:48.971415360 +0200
+@@ -686,8 +686,8 @@
+ #if !UCONFIG_NO_FILE_IO && ((U_PLATFORM_IS_DARWIN_BASED && (U_PLATFORM != U_PF_IPHONE || defined(U_TIMEZONE))) || U_PLATFORM_IS_LINUX_BASED || U_PLATFORM == U_PF_BSD || U_PLATFORM == U_PF_SOLARIS)
+ /* These platforms are likely to use Olson timezone IDs. */
+ /* common targets of the symbolic link at TZDEFAULT are:
+- * "/usr/share/zoneinfo/<olsonID>" default, older Linux distros, macOS to 10.12
+- * "../usr/share/zoneinfo/<olsonID>" newer Linux distros: Red Hat Enterprise Linux 7, Ubuntu 16, SuSe Linux 12
++ * "/usr/share/lib/zoneinfo/<olsonID>" default, Illumos / Solaris settings
++ * "../usr/share/lib/zoneinfo/<olsonID>" newer Linux distros: Red Hat Enterprise Linux 7, Ubuntu 16, SuSe Linux 12
+  * "/usr/share/lib/zoneinfo/<olsonID>" Solaris
+  * "../usr/share/lib/zoneinfo/<olsonID>" Solaris
+  * "/var/db/timezone/zoneinfo/<olsonID>" macOS 10.13
+@@ -705,12 +705,12 @@
+ #define TZ_ENV_CHECK    "localtime"
+ #else
+ #define TZDEFAULT       "/etc/localtime"
+-#define TZZONEINFO      "/usr/share/zoneinfo/"
++#define TZZONEINFO      "/usr/share/lib/zoneinfo/"
+ #endif
+ #define TZZONEINFOTAIL  "/zoneinfo/"
+ #if U_HAVE_DIRENT_H
+ #define TZFILE_SKIP     "posixrules" /* tz file to skip when searching. */
+-/* Some Linux distributions have 'localtime' in /usr/share/zoneinfo
++/* Some Linux distributions have 'localtime' in /usr/share/lib/zoneinfo
```

Comment:
OpenIndiana is not Linux :-).

Reply:
looking at the sheer number of changes i used sed :-). So unfortunately also comments where changed.